package.json

```diff
@@ -16,6 +16,7 @@
   "scripts": {
     "bundle": "gulp bundle",
     "dev": "gulp preview",
+    "devlive": "export LIVERELOAD=true; gulp preview",
```

Comment:
I suggest to add this command to the README to let people know it exists (we could mention the limitation explained in the pull request)

Reply:
I used the env var in the past on Ubuntu 20.04 and it didn't work as this is described in the PR. See #115.
I will redo so tests with this PR later.

Reply:
Agree, see my comment above